ctru-rs/src/services/mod.rs

```diff
@@ -18,6 +19,25 @@ mod reference;
 pub mod soc;
 pub mod sslc;
 
+cfg_if::cfg_if! {
+    if #[cfg(all(feature = "romfs", romfs_exists))] {
+        pub mod romfs;
+    } else {
+        pub mod romfs {
+            //! The RomFS folder has not been detected and/or the `romfs` feature has not been enabled.
+            //!
+            //! Configure the path in Cargo.toml (the default path is "romfs"). Paths are relative to the
+            //! `CARGO_MANIFEST_DIR` environment variable, which is the directory containing the manifest of
+            //! your package.
+            //!
+            //! ```toml
+            //! [package.metadata.cargo-3ds]
+            //! romfs_dir = "romfs"
+            //! ```
```

Comment:
at some point we might consider adding something like
```rs
} else if #[cfg(all(feature = "romfs", not(romfs_exists)))] {
    compile_error!("romfs feature is enabled but no romfs found!");
}
```

haven't tested to see if that does what I think though

Reply:
That's not really in the scope of this PR, but yes, ROMFS does have improvement potential.

Reply:
Yeah you're right, sorry. Opened #108